### ISSUE TITLE
SuperCollider can't boot on disabling microphone

### ISSUE THREAD

**Issue by jeiea** [supercollider/supercollider#844]
Error message is 
*PortAudio failed at Pa_OpenDefaultStream with error: 'Device unavailable' could not initialize audio.*
I found when disable microphone device on windows settings this error occurs.
Can it be fix to normally work on similar situation?

**Comment by sonoro1234** [comment]
https://github.com/supercollider/supercollider/commit/3f78951d722e1c936747e8e0e60a747ecb185f63
solved that?

**Comment by bagong** [comment]
I think it did! I tested on Thinkpad X201 with Win8.1 64-bit both with the inbuilt device and a gigaport HD+, a portable USB-soundcard with 8 outs and no in. I disabled the internal mic and everything went beautifully: with the Gigaport plugged in, the server would boot with MME : Speakers (GIGAPort HD+) only, with no in. Without  the Gigaport connected, the server would come up with the outs of the internal sound device. Even the opposite worked: with enabled mice and disabled sound out, the server would boot with In only. Lovely! Thank you...
